### DISCUSSION TITLE
Object Oriented Programming

### DISCUSSION THREAD

**Discussion by GUKWAT** [ryansurf/cli-surf#66] [Ideas]
Hi @ryansurf @K-dash I was thinking of adding classes to each python file in order for the code to be scalable what do you think?

**Comment by K-dash** [comment]
@GUKWAT (@ryansurf )
Thanks for the suggestion😊

You're right, most of our files except for `settings.py` are function-based modules.
If you don't mind, Could you elaborate on which specific parts you think would benefit most from being refactored into classes?

**Comment by python-monsters** [comment]
Hello everyone, I’m a computer science graduate student seeking real-world experiences in software engineering. I’d love to contribute to your project in any capacity — from bug fixes to feature development — in exchange for mentorship or experience. I’m familiar with Python, Java, C++, C#, SQL, and Git. Please let me  know if you have any projects or opportunities that match my skill set. I'm always up for challenge,eager to learn, hone my skills.

Thank you for your time,

Larry
